autogalaxy/config/priors/light/shapelets/exponential.yaml

```diff
@@ -0,0 +1,94 @@
+ShapeletExponential:
+  centre_0:
+    type: Gaussian
+    mean: 0.0
+    sigma: 0.3
+    lower_limit: -inf
+    upper_limit: inf

```

Comment:
gaussian limits are implicitly infinite

Reply:
Like to keep them there so users know they exist (I know this is source code but principle still applies).